src/status_im/contexts/shell/activity_center/events.cljs

```diff
@@ -408,6 +409,17 @@
                           (constantly processed)
                           #(concat % processed))))}))
 
+(re-frame/reg-fx :activity-center.notifications/fetch-pending-contact-requests-fx
```

Comment:
@alwx, this effect should go into a file named `effects.cljs`. The naming convention is to prepend with `:effects`. Hence, `:effects.activity-center/fetch-pending-contact-requests`.